cmd/snapd-apparmor/snapd-apparmor

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
```

Comment:
No `set -e` for the script? If so, could you please document why this is omitted

Reply:
No because this mimics the behaviour of the distribution script that carries on even if things fail. I will document it better.